ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Protocol.hs

```diff
@@ -454,7 +454,7 @@ instance TPraosCrypto c => ConsensusProtocol (TPraos c) where
       rho = VRF.evalCertified () rho' tpraosIsCoreNodeSignKeyVRF
       y   = VRF.evalCertified () y'   tpraosIsCoreNodeSignKeyVRF
 
-      tickEta0 (STS.TicknState _ x) = x
+      tickEta0 (STS.TicknState x _) = x
```

Comment:
A good reason to use the field (`ticknStateEpochNonce`) :grin:

Reply:
Yes, indeed!

Reply:
In fact, let's just do that...